jenkins-slaves/ansible/roles/kogito/tasks/main.yml

```diff
@@ -0,0 +1,11 @@
+---
+### Install NodeJS package ###
+
+- name: Install NodeJS package
+  yum:
+    pkg: 'nodejs'
+    state: latest
+
+- name: Install Yarn
+  shell: npm install -g yarn
```

Comment:
You probably need to add `--registry=${NPM_REGISTRY_URL}` since we do not have global `.npmrc` file yet